tests/stress_tests/scripts/compare_memcheck_2_runs.py

```diff
@@ -165,13 +165,18 @@ def cli_parser():
                         help='Source of reference values of MemCheckTests. '
                              'Should contain path to a folder with logs or '
                              'JSON-format query to request data from DB.')
+    args = parser.parse_known_args()[0]
```

Comment:
This will break --help output. Please refer to run_memcheck.py on how to fix it:

init_parser = argparse.ArgumentParser(**add_help=False**)
...

args = init_parser.parse_known_args()[0]

parser = argparse.ArgumentParser(**parents=[init_parser]**)

Reply:
@asomsiko --help was really broken. But provided solution can't be applied because `parse_known_args` works correctly for optional arguments, but positional arguments still treated as required. So I added checks outside of `argparse`